[12.0][ADD]hr_gamification_email_notification (#869)
OCA/hr: hr_gamification_email_notification/readme/DESCRIPTION.rst

> This module Sends Email to all the Employees with all the award given in
> the last week.

Reviewer: ```suggestion
This module sends an email every week to all the employees with all the badges awarded during
the previous week.
```
